python/test/unit/language/test_core.py

```diff
@@ -7130,6 +7144,7 @@ def inject_layout(ir, src: torch.Tensor, axis, indices: torch.Tensor, src_layout
     output, compiled = prepare_kernel(src, axis, indices)
     ir = compiled.asm["ttgir"]
     ir = inject_layout(ir, src, axis, indices, src_layout, indices_layout)
+    assert ir != compiled.asm["ttgir"]
```

Comment:
Added this line to ensure that IR was changed.
Because of changes in tt.gather this test did not test given layouts, but default one, provided by `gather_test_kernel`